src/Quiz.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import './App.css';
+import Demon_sprite from './data/spr/Demon_sprite.png'
+import Imp_sprite from './data/spr/Imp_sprite.png'
+import useSound from 'use-sound';
+import dssgtatk from './data/snd/dssgtatk.wav';
+// /static/media/dssgtatk.cd1a3ad1.wav
+
+function Quiz(props) {
+  // const props = {
```

Comment:
Add an additional comment with title to understand what is this exactly

Reply:
Changed the code.